tests_e2e/orchestrator/scripts/install-agent

```diff
@@ -140,6 +140,24 @@ if [[ $(uname -a) == *"flatcar"* ]]; then
   if [[ ! -f /usr/share/oem/waagent.conf ]]; then
     ln -s "$waagent_conf_path" /usr/share/oem/waagent.conf
   fi
+
```

Comment:
let's move this code to the if statement for flatcar

Reply:
This code already inside if statement for flatcar